includes/Core/Modules/Modules.php

```diff
@@ -816,6 +825,15 @@ private function prepare_module_data_for_response( Module $module ) {
 			'dependencies' => $this->get_module_dependencies( $module->slug ),
 			'dependants'   => $this->get_module_dependants( $module->slug ),
 		);
+
+		if ( $module instanceof Module_With_Owner ) {
+			$owner_id = $module->get_owner_id();
+			if ( $owner_id ) {
+				$module_data['owner'] = get_the_author_meta( 'display_name', $owner_id );
```

Comment:
Per the ACs, this should use the username (kind of the "user slug"), not the display name. Furthermore, I think it would make more sense to alter the data format here:

* The REST API endpoints should be future proof. Exposing a name as something like `owner` can quickly get us trapped in a place where we don't have enough flexibility to change the format later - for example, it's not clear whether `owner` refers to an ID, slug, display name, ... I think we should make `owner` an object (associative array) here and expose `id` and `login` on it for now.
* API response data should be consistent, there shouldn't be any properties that exist on some responses but are missing somewhere else. As such, if there is no owner, or if the module is not a `Module_With_Owner`, this should still be included, as `null`.

Reply:
Good point. Updated.